pandas/tests/io/json/test_pandas.py

```diff
@@ -1032,6 +1038,102 @@ def test_to_jsonl(self):
         assert result == expected
         assert_frame_equal(pd.read_json(result, lines=True), df)
 
+    def test_readjson_chunks(self, lines_json_df):
+        """Basic test that read_json(chunks=True) gives the same result as
+        read_json(chunks=False)"""
+        # GH17048: memory usage when lines=True
+
+        for cs in [1, 1.0]:
+
+            unchunked = pd.read_json(StringIO(lines_json_df), lines=True)
```

Comment:
if yes to the above then parametrize this test with cs (but call it chunksize)

Reply:
done